tests/Functional/Action/WebHook/UpdateLineItemsWebhookActionTest.php

```diff
@@ -108,7 +142,10 @@ public function testItAcceptLineItemsToBeUpdated(): void
             '/api/v1/web-hooks/update-line-items',
             [],
             [],
-            [],
+            [
+                'PHP_AUTH_USER' => 'testUsername',
+                'PHP_AUTH_PW' => 'testPassword'
```

Comment:
Same as previous (about credentials in config)

Reply:
Please see https://github.com/oat-sa/simple-roster/pull/182#discussion_r551911064